Library/DiscUtils.Ext/VfsExtFileSystem.cs

```diff
@@ -73,13 +73,13 @@ public VfsExtFileSystem(Stream stream, FileSystemParameters parameters)
             long blockDescStart = (superblock.FirstDataBlock + 1) * (long)superblock.BlockSize;
 
             stream.Position = blockDescStart;
-            var bgDescSize = superblock.Has64Bit ? BlockGroup64.DescriptorSize64 : BlockGroup.DescriptorSize;
+            var bgDescSize = superblock.Has64Bit ? superblock.DescriptorSize : BlockGroup.DescriptorSize;
             byte[] blockDescData = StreamUtilities.ReadExact(stream, (int)numGroups * bgDescSize);
 
             _blockGroups = new BlockGroup[numGroups];
             for (int i = 0; i < numGroups; ++i)
             {
-                BlockGroup bg = superblock.Has64Bit ? new BlockGroup64() : new BlockGroup();
+                BlockGroup bg = superblock.Has64Bit ? new BlockGroup64(bgDescSize) : new BlockGroup();
```

Comment:
A bit like above, would it make sense to merge `BlockGroup` and `BlockGroup64` ?

Reply:
Yes, I did that with the latest commit. Please take a look.

Reply:
Wow. I'm so sorry, I didn't realize BlockGroup64 inherited BlockGroup. It's best to leave them be then.

Reply:
I'm unable to revert your commit on your branch. :/

Reply:
I removed the latest commit.